.gitignore

```diff
@@ -26,3 +26,6 @@ _output/
 # Output of the go coverage tool, specifically when used with LiteIDE
 coverage.txt
 *.out
+
+# VSCode editor files
+.vscode/
```

Comment:
As mentioned in the `NOTE` section on top of this file, we should not include editor-specific stuff in this `.gitignore` file?